sys/event/event.c

```diff
@@ -38,9 +38,10 @@ void event_post(event_queue_t *queue, event_t *event)
     assert(queue && event);
 
     unsigned state = irq_disable();
-    if (!event->list_node.next) {
+    if (event->num_posted == 0) {
         clist_rpush(&queue->event_list, &event->list_node);
     }
+    ++event->num_posted;
```

Comment:
This needs to be an atomic read-modify-write sequency

Reply:
I guess I should have looked at the whole implementation, not at the diff the web UI is showing only (=